openssl-dynamic/src/main/c/sslcontext.c

```diff
@@ -244,17 +245,19 @@ TCN_IMPLEMENT_CALL(jlong, SSLContext, make)(TCN_STDARGS, jint protocol, jint mod
         goto cleanup;
     }
 
-    TCN_THROW_IF_ERR(apr_pool_create(&p, tcn_global_pool), p);
-
-    if ((c = apr_pcalloc(p, sizeof(tcn_ssl_ctxt_t))) == NULL) {
-        tcn_ThrowAPRException(e, apr_get_os_error());
+    if ((c = calloc(1, sizeof(tcn_ssl_ctxt_t))) == NULL) {
+        tcn_Throw(e, "Failed to calloc tcn_ssl_ctxt_t");
         goto cleanup;
     }
 
     c->protocol = protocol;
     c->mode     = mode;
     c->ctx      = ctx;
-    c->pool     = p;
+    c->ticket_keys = tcn_atomic_uint32_new();
```

Comment:
Was the intention that this should be `ticket_keys_new` and `ticket_keys` should be initialized in some other way? It is freed above using `OPENSSL_free`.